example/.flutter-plugins-dependencies

```diff
@@ -1 +1 @@
-{"_info":"// This is a generated file; do not edit or check into version control.","dependencyGraph":[{"name":"flutter_stegify","dependencies":[]},{"name":"path_provider","dependencies":[]}]}
\ No newline at end of file
+{"info":"This is a generated file; do not edit or check into version control.","plugins":{"ios":[{"name":"flutter_stegify","path":"/Users/sabin/AndroidStudioProjects/stegify-flutter-plugin/","dependencies":[]},{"name":"path_provider","path":"/Users/sabin/flutter/.pub-cache/hosted/pub.dartlang.org/path_provider-1.6.7/","dependencies":[]}],"android":[{"name":"flutter_stegify","path":"/Users/sabin/AndroidStudioProjects/stegify-flutter-plugin/","dependencies":[]},{"name":"path_provider","path":"/Users/sabin/flutter/.pub-cache/hosted/pub.dartlang.org/path_provider-1.6.7/","dependencies":[]}],"macos":[{"name":"path_provider_macos","path":"/Users/sabin/flutter/.pub-cache/hosted/pub.dartlang.org/path_provider_macos-0.0.4+2/","dependencies":[]}],"linux":[],"windows":[],"web":[]},"dependencyGraph":[{"name":"flutter_stegify","dependencies":[]},{"name":"path_provider","dependencies":["path_provider_macos"]},{"name":"path_provider_macos","dependencies":[]}],"date_created":"2020-05-09 22:53:07.506355","version":"1.17.0"}
```

Comment:
This file contains paths from your machine and is marked as one that should not be checked into a VCS, can you please remove this file from the PR, since it is specific for your local setup.

Reply:
sure

Reply:
hello there I have removed
` example/.flutter-plugins-dependencies`
from PR and also added to `.gitignore` also